<a href="https://colab.research.google.com/github/SovetovAleksey/Probability-theory/blob/5_quest/5.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

In [1]:
import numpy as np

# 1. Известно, что генеральная совокупность распределена нормально со средним квадратическим отклонением, равным 16. Найти доверительный интервал для оценки математического ожидания a с надежностью 0.95, если выборочная средняя M = 80, а объем выборки n = 256.

$$σ = 16$$
$$\alpha = 0.05$$
$$M = 80$$
$$n = 256$$
$$t_{1- \alpha/2} = 1.96$$

$$T_{1,2} = \overline{X} \pm \frac{s_0}{\sqrt{n}} \cdot t_{1 - \alpha/2, n - 1} = 80 \pm \frac{16}{\sqrt{256}} \cdot 1.96 = 80 \pm 1.96$$



# 2. В результате 10 независимых измерений некоторой величины X, выполненных с одинаковой точностью, получены опытные данные: 6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1 Предполагая, что результаты измерений подчинены нормальному закону распределения вероятностей, оценить истинное значение величины X при помощи доверительного интервала, покрывающего это значение с доверительной вероятностью 0,95.

$\alpha = 0.05, n - 1= 9, T_{1,2} = \overline{X} \pm \frac{s_0}{\sqrt{n}} \cdot c_{\gamma}, t_{1 - \alpha/2, n - 1} = 2.2622$

In [13]:
array = np.array([6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1])
array

array([6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1])

In [7]:
array_mean = array.mean()
array_mean

6.590000000000001

In [9]:
array_std = array.std(ddof = 1)
array_std

0.4508017549014448

$$T_{1,2} = \overline{X} \pm \frac{s_0}{\sqrt{n}} \cdot t_{1 - \alpha/2, n - 1} = 6.59 \pm \frac{0.4508017549014448}{\sqrt{10}} \cdot 2.2622 = 6.59 \pm 0,3224902554$$

# 3. Утверждается, что шарики для подшипников, изготовленные автоматическим станком, имеют средний диаметр 17 мм. Используя односторонний критерий с α=0,05, проверить эту гипотезу, если в выборке из n=100 шариков средний диаметр оказался равным 17.5 мм, а дисперсия известна и равна 4 мм.

$$α = 0,05$$
$$n = 100$$
$$D(X) = 4$$
$$H0: 17,5 = 17$$
$$H1: 17,5 \ne 17$$

$$σ = \sqrt{D(X)} = \sqrt{4} = 2$$
$$Z_{H} = \dfrac{\overline{X} - m}{\sigma_X / \sqrt{n}} = \frac{17,5 - 17}{2/\sqrt{100}} = \frac{0.5}{0.2} = 2.5$$

$$t_{1 - \alpha/2, n - 1} = 1.9842$$

$$Z_{H} > t_{1 - \alpha/2, n - 1}$$

$$2.5 > 1.9842$$




Отвергаем нулевую гипотезу, принимаем альтернативную гипотезу

# 4. Продавец утверждает, что средний вес пачки печенья составляет 200 г. Из партии извлечена выборка из 10 пачек. Вес каждой пачки составляет: 202, 203, 199, 197, 195, 201, 200, 204, 194, 190. Известно, что их веса распределены нормально. Верно ли утверждение продавца, если учитывать, что доверительная вероятность равна 99%?

$$α = 0.01, n = 10$$

In [10]:
array = np.array([202, 203, 199, 197, 195, 201, 200, 204, 194, 190])

In [11]:
array_mean = array.mean()
array_mean

198.5

$$H0: 200 = 198,5$$
$$H1: 200 \ne 108,5$$

In [12]:
array_std = array.std(ddof = 1)
array_std

4.453463071962462

$$Z_{H} = \dfrac{\overline{X} - m}{\sigma_X / \sqrt{n}} = \frac{198,5 - 200}{4.453463071962462/\sqrt{10}} = \frac{-1.5}{1.4083086783} = -1.0651074037$$

$$t_{1 - \alpha/2, n - 1} = 2,7638$$

$$Z_{H} \in [-2,7638; +2,7638]$$

Нулевая гипотеза верна